include/linux/tee_drv.h

```diff
@@ -586,6 +593,48 @@ static inline bool tee_param_is_memref(struct tee_param *param)
 	}
 }
 
+static inline bool tee_ioctl_param_is_ocall_reply(struct tee_ioctl_param *param)
```

Comment:
These `tee_ioctl_params_is*()` can be moved into `drivers/tee/tee_core.c`

Reply:
Moved.